spec/miscellaneous/lib/tasks/after_tasks.rake_spec.rb

```diff
@@ -97,3 +97,61 @@
     end
   end
 end
+
+describe "rake After:add_user_id_to_kudos" do
+  context "when the kudos has no pseud_id" do
+    it "doesn't add user_id if the kudos has an ip_address" do
+      guest_kudos = create(:kudo, ip_address: "0.00.0.000")
```

Comment:
If we are making changes could we change the ip address to 127.0.0.2  or at least something valid, its not important but its going to annoy me to have this string.

Reply:
We don't validate IP addresses, so it shouldn't matter, but sure! I'll fix the others, too.